generated/nifgen/nifgen/_library.py

```diff
@@ -387,21 +387,21 @@ def niFgen_GetHardwareState(self, vi, state):  # noqa: N802
                 self.niFgen_GetHardwareState_cfunc.restype = ViStatus  # noqa: F405
         return self.niFgen_GetHardwareState_cfunc(vi, state)
 
-    def niFgen_GetLastExtCalLastDateAndTime(self, vi, month):  # noqa: N802
+    def niFgen_GetLastExtCalLastDateAndTime(self, vi, last_cal_datetime):  # noqa: N802
```

Comment:
There is something wrong in our code generator that it creates this wrapper function for something that doesn't have a C function. @sbethur do we have any easy way to tell that the functions shouldn't be getting generated? `'python-only'`?

Reply:
Yes, we should use 'python-only`, I'll fix it.

Reply:
Fixed it in internal metadata. @ayesham-729, please update this PR with the latest export of internal metadata.

Reply:
Updated